.rubocop.yml

```diff
@@ -30,6 +30,10 @@ AllCops:
 Metrics/MethodLength:
   Description: 'Avoid methods longer than 30 lines of code.'
   Max: 30 # default is 10
+  CountAsOne:
+    - array
+    - hash
+    - heredoc
```

Comment:
I've always thought that `Metrics/MethodLength` was too literal in terms of how it counts lines of code.  I discovered that you can tell it to count arrays/hashes/etc as a single line.  I think this is a good default for Katello going forward.

Reply:
```
Lint/RedundantCopDisableDirective: Unnecessary disabling of Metrics/MethodLength
```
looks like I may have to extract this to another PR :cry:

Reply:
Opened https://github.com/Katello/katello/pull/9231

Reply:
This should be removed now since you've got an open PR :scissors: